ddl/ddl_api.go

```diff
@@ -885,6 +885,13 @@ func (d *ddl) AlterTable(ctx context.Context, ident ast.Ident, specs []*ast.Alte
 			err = d.RenameTable(ctx, ident, newIdent)
 		case ast.AlterTableDropPrimaryKey:
 			err = ErrUnsupportedModifyPrimaryKey.GenByArgs("drop")
+		case ast.AlterTableOption:
+			for _, opt := range spec.Options {
+				if opt.Tp == ast.TableOptionAutoIncrement {
+					err = d.RebaseAutoID(ctx, ident, int64(opt.UintValue-1))
```

Comment:
Why this is opt.UintValue-1?

Reply:
It is compatible with MySQL, if we set auto_increment = 10, the next auto increment insert column should take 10.

Reply:
int64(opt.UintValue)-1 should be better. I'll fix it.